UHF/simulated_uhf.py

```diff
@@ -38,39 +52,69 @@ def check_port(port):
         try:
             s.bind(("0.0.0.0", port))
             return True
+
         except OSError:
             return False
 
 
-def beacon(client, lock):
+def mod_beacon(message, lock):
```

Comment:
great work! Now I think the next step is to have the beacon period be modifiable.